src/onegov/org/models/extensions.py

```diff
@@ -999,3 +999,31 @@ def links_to_json(
                 })
 
         return SidebarLinksForm
+
+
+class DeletableContentExtension(ContentExtension):
+    """ Extends any class that has a meta dictionary field with the ability to
+    mark the content as deletable after reaching the end date. A cronjob will
+    periodically check for 'deletable' content with expired end date and
+    delete it e.g. Directories.
+
+    """
+
+    delete_when_expired: dict_property[bool] = content_property(default=False)
+
+    # according which field the content should be deleted?
+    # class UTCPublicationMixin
+
```

Comment:
Are these comments still relevant? Else remove

Reply:
going to remove